extensions/amp-story/1.0/amp-story-page.js

```diff
@@ -1659,4 +1683,38 @@ export class AmpStoryPage extends AMP.BaseElement {
       );
     });
   }
+
+  /**
+   * Listens for changes on captions if there are tracks on videos and page is active.
+   * @private
+   */
+  initializeCaptionsListener_() {
+    if (!this.element.querySelector('track')) {
+      return;
+    }
```

Comment:
I wonder how this will work when captions are added dynamically with Monti.
Maybe we will need to make this public and call it when a track is appended?

Reply:
I pushed a change some time ago (Github seems to be lagging) that waits for the video elements to resolve the video cache and _then_ checks this. When Github gets fixed, the commit should show up that would fix that.

Reply:
Awesome! I see it now, looks like it's happening here:
https://github.com/ampproject/amphtml/pull/37884/files#diff-94f9e2fa2969f98d22c37d9e1f195adbe79ebda93b4ffedf4eab34327238418fR609-R611